src/config.py

```diff
@@ -20,9 +20,14 @@ def sha256(hashable) -> str:
 
 PORTS = SimpleNamespace(
     LOKI_HTTP=3500,
+    OTLP_GRPC=4317,
     OTLP_HTTP=4318,
     METRICS=8888,
     HEALTH=13133,
+    # Tracing
+    JAEGER_GRPC=14250,
+    JAEGER_THRIFT_HTTP=14268,
+    ZIPKIN=9411,
 )
```

Comment:
Should we add a utest that the port values are unique?

Reply:
Done